translate/translate.py

```diff
@@ -14,57 +15,68 @@ class Translate(commands.Cog):
     def __init__(self, bot: Red, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.bot = bot
-        self.settings = TranslateSettings("translate")
+        self.config = Config.get_conf(self, identifier=724757473)
+        self.config.register_global(api_key=None)
 
         self.service = None
-        self.trySetupService()
 
-    def trySetupService(self):
-        api_key = self.settings.get_key()
-        if api_key:
-            self.service = build('translate', 'v2', developerKey=api_key)
 
     @commands.group()
     @checks.is_owner()
     async def translate(self, context):
         """Translation utilities."""
 
+
+    @translate.command()
+    async def build_service(self):
+        api_key = await self.config.api_key()
+        try:
+            assert api_key
+            self.service = build('translate', 'v2', developerKey=api_key)
+        except:
```

Comment:
FYI it's bad practice to use assertions and exception handling for control flow. An if/else would even be less lines of code.